src/NUnitFramework/tests/Internal/TestExecutionContextTests.cs

```diff
@@ -581,9 +581,12 @@ public void CanAccessParallelScope()
         [Test]
         public void CanAccessTestWorker()
         {
-            Assert.That(_fixtureContext.TestWorker, Is.Not.Null);
-            Assert.That(_setupContext.TestWorker, Is.SameAs(_fixtureContext.TestWorker));
-            Assert.That(TestExecutionContext.CurrentContext.TestWorker, Is.SameAs(_setupContext.TestWorker));
+            if (TestExecutionContext.CurrentContext.Dispatcher is ParallelWorkItemDispatcher)
+            {
+                Assert.That(_fixtureContext.TestWorker, Is.Not.Null);
+                Assert.That(_setupContext.TestWorker, Is.SameAs(_fixtureContext.TestWorker));
+                Assert.That(TestExecutionContext.CurrentContext.TestWorker, Is.SameAs(_setupContext.TestWorker));
+            }
```

Comment:
Can you clarify this change? Am I right that this will mean this test will now be empty, if we ever run the tests in serial? I'm a little confused as to what's changed that requires this, sorry?